Tests/Functional/RunCommandTest.php

```diff
@@ -14,20 +14,20 @@ class RunCommandTest extends BaseTestCase
 
     public function testRun()
     {
-        $a = new Job('a');
```

Comment:
This is confusing for the application, since it has 2 commands starting with an `a`: `about` and `assets:install`